tools/refresh-keyrings.sh

```diff
@@ -46,10 +50,6 @@ ESM_KEY_ID_BIONIC="2926E7D347A1955504000A983121D2531EF59819"
 FIPS_KEY_ID_XENIAL="E23341B2A1467EDBF07057D6C1997C40EDE22758"
 
 
-UA_KEYRING="ubuntu-advantage-keyring.gpg"
-
-generate_keyring $TARGET_DIR/$UA_KEYRING $EAL_KEY_ID_XENIAL $ESM_KEY_ID_PRECISE $ESM_KEY_ID_TRUSTY $ESM_KEY_ID_XENIAL $ESM_KEY_ID_BIONIC $FIPS_KEY_ID_XENIAL
+generate_keyrings $TARGET_DIR $EAL_KEY_ID_XENIAL $ESM_KEY_ID_PRECISE $ESM_KEY_ID_TRUSTY $ESM_KEY_ID_XENIAL $ESM_KEY_ID_BIONIC $FIPS_KEY_ID_XENIAL
 sed -i "s/ESM_KEY_ID_TRUSTY=.*/ESM_KEY_ID_TRUSTY=\"${ESM_KEY_ID_TRUSTY}\"/" \
-    -i "s/ESM_KEY_ID_XENIAL=.*/ESM_KEY_ID_XENIAL=\"${ESM_KEY_ID_XENIAL}\"/" \
```

Comment:
Why are these lines being dropped?

Reply:
I'll add a comment in the code. These sed lines are dropped because we aren't pre-seeding any ESM keys in debian/postinst (because we only manipulate esm keys on trusty and pre-install keys for esm package visibility). All other keys are added/removed only and runtime during ua enable (or auto-enablement on attach)